RELEASE_NOTES.md

```diff
@@ -4,6 +4,11 @@ This page describes the noteworthy improvements provided by each release of Ecli
 
 ## 3.0.0 (under development)
 
+### Support for Products with mixed Features and Plugins
+
+Tycho now supports building _mixed_ Products, which have features and plug-ins listed and get both installed.
+Furthermore Product attribute 'type' is now supported, which can have the values 'bundles', 'features' and 'mixed' and supersedes the boolean-valued 'useFeatures' attribute.
```

Comment:
Do you mean `type` it takes precedence over `useFeatures`?
`supersedes` seem to indicate I cannot use `useFeatures` anymore.

Reply:
> Do you mean `type` it takes precedence over `useFeatures`? `supersedes` seem to indicate I cannot use `useFeatures` anymore.

Yes I do. Adjusted the test accordingly.